docker/addons/prometheus/grafana/example-dashboard.json

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
For total request can you implement something like this?

https://play.grafana.org/d/000000029/prometheus-demo-dashboard?orgId=1&refresh=5m

![image](https://user-images.githubusercontent.com/28790446/231512818-1ab55780-539b-432f-87bc-5680208f121f.png)

Reply:
Still not fixed